src/libterm/terminfo/parser/compiled.rs

```diff
@@ -169,6 +169,18 @@ fn read_le_u16(r: &mut dyn io::Read) -> io::Result<u16> {
     Ok((b[0] as u16) | ((b[1] as u16) << 8))
 }
 
+fn read_le_u32(r: &mut dyn io::Read) -> io::Result<u32> {
+    let mut b = [0; 4];
+    let mut amt = 0;
+    while amt < b.len() {
```

Comment:
We should be able to use `read_exact(&mut b)` here.

Reply:
Done.